data/templates/install-photon/photon-os-ks

```diff
@@ -29,14 +29,14 @@
                         <%_ numSize = 0; _%>
                     <%_ } _%>
                     <%_ if (par.fsType === "swap") { _%>
-                        <%_ if (i === length ) { _%>
-        {"size": <%=numSize%>, "filesystem": "<%=par.fsType%>"}
-                        <%_ } else { _%>
-        {"size": <%=numSize%>, "filesystem": "<%=par.fsType%>"},
-                        <%_ } _%>
+                        <%_ swapPar = par; _%>
+                        <%_ swapParSize = numSize; _%>
                     <%_ } else { _%>
-                        <%_ if (i === length ) { _%>
+                        <%_ if (i === length && typeof swapPar === 'undefined') { _%>
         {"mountpoint": "<%=par.mountPoint%>", "size": <%=numSize%>, "filesystem": "<%=par.fsType%>"}
+                        <%_ } else if (i === length && typeof swapPar !== 'undefined') { _%>
+        {"mountpoint": "<%=par.mountPoint%>", "size": <%=numSize%>, "filesystem": "<%=par.fsType%>"},
```

Comment:
Please add some comment so that other developer will be much more easier to understand such kind of special design. 
Ejs has a special tag to add comment.